doc/customTags.hbs

```diff
@@ -0,0 +1,3 @@
+{{#if customTags}}{{#each customTags}}{{#unless (isAsync)}}**{{titleCase tag}}**: {{#with (parseType value)~}}
```

Comment:
Unfortunately, filenames are fixed too: https://github.com/jsdoc2md/dmd#write-a-new-main-partial
`the file basename of a partial is significant - if you wish to override main (invoked by {{>main}}) then the filename of your partial must be main.hbs.`